app/src/main/java/br/com/dionataferraz/vendas/account/data/local/AccountDatabase.kt

```diff
@@ -0,0 +1,28 @@
+package br.com.dionataferraz.vendas.account.data.local
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import br.com.dionataferraz.vendas.account.converters.Converters
+
+@Database(entities = [AccountEntity::class], version = 1)
+@TypeConverters(Converters::class)
+abstract class AccountDatabase: RoomDatabase() {
+
+    abstract fun AccDao(): AccountDao
```

Comment:
E para acessar os DAOs assim:
```suggestion
            abstract fun accDao(): AccountDao
            abstract fun userDao(): UserDao
            abstract fun transactionDao(): TransactionDao
```